cdc/processor/sinkmanager/mem_quota.go

```diff
@@ -78,14 +78,18 @@ func (m *memQuota) forceAcquire(nBytes uint64) {
 }
 
 // blockAcquire is used to block the request when the memory quota is not available.
-func (m *memQuota) blockAcquire(nBytes uint64) error {
+func (m *memQuota) blockAcquire(nBytes uint64, isAborted func() bool) error {
 	m.mu.Lock()
 	defer m.mu.Unlock()
 	for {
 		if m.isClosed.Load() {
 			return cerrors.ErrFlowControllerAborted.GenWithStackByArgs()
 		}
 
+		if isAborted() {
```

Comment:
It's possible that here it's not aborted, but it becomes aborted immediately. Is it ok?